.changeset/thin-ligers-fold.md

```diff
@@ -0,0 +1,6 @@
+---
+"@firebase/firestore": patch
+"firebase": patch
+---
+
+Fixed an issue where `onSnapshot` doesn't work with rxjs [Observer](https://rxjs-dev.firebaseapp.com/guide/observer).
```

Comment:
FWIW, the change that broke this was merged on Friday and never released.

Reply:
which one is it?

Reply:
https://github.com/firebase/firebase-js-sdk/pull/3294/files#diff-9cfbf72c75b0e1f9df5069543a6bced1R1172

Reply:
Cool. Changed to an empty changeset.